### ISSUE TITLE
Thanks for the token

### ISSUE THREAD

**Issue by Laddervr** [gytisfr/Ratio-Bot#1]
You forgot to remove your token

**Comment by gytisfr** [comment]
yeah I know, I put up a ton of old projects today and wasn’t bothered to remove any tokens because all of the bots have been deleted now, thanks for looking out though
